File: week 3/change/change.py
def get_change(n):
    #write your code here
    result = 0
    result += int(n/10)
    n = n % 10
    result += int(n/5)
    n = n % 5
    n = n + result
    return n

if __name__ == '__main__':
    n = int(input())
    print(get_change(n))

File: week 3/change/Change.java
import java.util.Scanner;

public class Change {
    private static int getChange(int n) {
        //write your code here
		int result = 0;
		result += n/10;
		n = n % 10;
		result += n/5;
		n = n % 5;
		n = n + result;
        return n;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        System.out.println(getChange(n));

    }
}